library/gradle

```diff
@@ -6,49 +6,64 @@ GitRepo: https://github.com/keeganwitt/docker-gradle.git
+GitCommit: 8d22e033a49acb24572502355618cfa3dbb345bd
 Directory: jdk8
 
 Tags: 8.4.0-jdk8-focal, 8.4-jdk8-focal, 8-jdk8-focal, jdk8-focal
 Architectures: amd64, arm32v7, arm64v8, ppc64le
-GitCommit: ae8311e2e41fbd2837f7a39df7e4b24e00f9b905
+GitCommit: 8d22e033a49acb24572502355618cfa3dbb345bd
 Directory: jdk8-focal
 
 Tags: 8.4.0-jdk11, 8.4-jdk11, 8-jdk11, jdk11, 8.4.0-jdk11-jammy, 8.4-jdk11-jammy, 8-jdk11-jammy, jdk11-jammy
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: ae8311e2e41fbd2837f7a39df7e4b24e00f9b905
+GitCommit: 8d22e033a49acb24572502355618cfa3dbb345bd
 Directory: jdk11
 
 Tags: 8.4.0-jdk11-focal, 8.4-jdk11-focal, 8-jdk11-focal, jdk11-focal
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: ae8311e2e41fbd2837f7a39df7e4b24e00f9b905
+GitCommit: 8d22e033a49acb24572502355618cfa3dbb345bd
 Directory: jdk11-focal
 
 Tags: 8.4.0-jdk11-alpine, 8.4-jdk11-alpine, 8-jdk11-alpine, jdk11-alpine
 Architectures: amd64
-GitCommit: ae8311e2e41fbd2837f7a39df7e4b24e00f9b905
+GitCommit: 8d22e033a49acb24572502355618cfa3dbb345bd
 Directory: jdk11-alpine
 
 Tags: 8.4.0-jdk17, 8.4-jdk17, 8-jdk17, jdk17, 8.4.0-jdk, 8.4-jdk, 8-jdk, jdk, 8.4.0, 8.4, 8, latest, 8.4.0-jdk17-jammy, 8.4-jdk17-jammy, 8-jdk17-jammy, jdk17-jammy, 8.4.0-jdk-jammy, 8.4-jdk-jammy, 8-jdk-jammy, jdk-jammy, 8.4.0-jammy, 8.4-jammy, 8-jammy, jammy
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: ae8311e2e41fbd2837f7a39df7e4b24e00f9b905
+GitCommit: 8d22e033a49acb24572502355618cfa3dbb345bd
 Directory: jdk17
 
 Tags: 8.4.0-jdk17-focal, 8.4-jdk17-focal, 8-jdk17-focal, jdk17-focal, 8.4.0-jdk-focal, 8.4-jdk-focal, 8-jdk-focal, jdk-focal, 8.4.0-focal, 8.4-focal, 8-focal, focal
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: ae8311e2e41fbd2837f7a39df7e4b24e00f9b905
+GitCommit: 8d22e033a49acb24572502355618cfa3dbb345bd
 Directory: jdk17-focal
 
 Tags: 8.4.0-jdk17-alpine, 8.4-jdk17-alpine, 8-jdk17-alpine, jdk17-alpine, 8.4.0-jdk-alpine, 8.4-jdk-alpine, 8-jdk-alpine, jdk-alpine, 8.4.0-alpine, 8.4-alpine, 8-alpine, alpine
 Architectures: amd64
-GitCommit: ae8311e2e41fbd2837f7a39df7e4b24e00f9b905
+GitCommit: 8d22e033a49acb24572502355618cfa3dbb345bd
 Directory: jdk17-alpine
 
 Tags: 8.4.0-jdk17-graal, 8.4-jdk17-graal, 8-jdk17-graal, jdk17-graal, 8.4.0-jdk-graal, 8.4-jdk-graal, 8-jdk-graal, jdk-graal, 8.4.0-graal, 8.4-graal, 8-graal, graal, 8.4.0-jdk17-graal-jammy, 8.4-jdk17-graal-jammy, 8-jdk17-graal-jammy, jdk17-graal-jammy, 8.4.0-jdk-graal-jammy, 8.4-jdk-graal-jammy, 8-jdk-graal-jammy, jdk-graal-jammy, 8.4.0-graal-jammy, 8.4-graal-jammy, 8-graal-jammy, graal-jammy
 Architectures: amd64, arm64v8
-GitCommit: ae8311e2e41fbd2837f7a39df7e4b24e00f9b905
+GitCommit: 8d22e033a49acb24572502355618cfa3dbb345bd
 Directory: jdk17-graal
 
+Tags: 8.4.0-jdk21, 8.4-jdk21, 8-jdk21, jdk21, 8.4.0-jdk21-jammy, 8.4-jdk21-jammy, 8-jdk21-jammy, jdk21-jammy
+Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
```

Comment:
Are all these architectures supported by the temurin image?
https://github.com/adoptium/containers/blob/7360285847f9a281dfb23682be1351482a5ebe58/21/jdk/ubuntu/jammy/Dockerfile.releases.full

Reply:
Ah, no (at least not yet). I'd copy/pasted from 17. Thanks for pointing out!